packages/fxa-content-server/app/scripts/templates/choose_what_to_sync.mustache

```diff
@@ -9,7 +9,7 @@
           {{#t}}Choose what to sync:{{/t}}
         </h1>
       </header>
-      <div class="flex flex-wrap ltr:text-left rtl:text-right ltr:mobileLandscape:ml-6 rtl:mobileLandscape:mr-6">
+      <div class="flex flex-wrap text-start ltr:mobileLandscape:ml-6 rtl:mobileLandscape:mr-6">
```

Comment:
Too bad I can't find a "margin-start" style thing in Tailwind&hellip;

Although I do find the https://github.com/20lives/tailwindcss-rtl plugin rather interesting. (via https://github.com/tailwindlabs/tailwindcss/discussions/1483)
Curious if that'd let us do something like `mobileLandscape:ms-6` and shrink a bunch of our Tailwind rtl boilerplate.

Reply:
@pdehaan Haha, that would just be too good to be true. Heh, yeah that package looks interesting. I wonder if it has all the stuff we'd need and if it'd be worth the refactor, idk if it would be unless it helped quite a lot. It seems to have a slightly higher number of downloads than what we're using ([tailwindcss-dir](https://github.com/RonMelkhior/tailwindcss-dir)) though, IIRC we went with `dir` since there weren't many options at the time and a contributor actually got it started for us.